Aptoide/aptoide-client-v8
# [Task/APP-79] Remove appc upgrades section

**What does this PR do?**

   Removes the AppCoins upgrades section from Apps (we had performance issues affecting our services)
   
   This still supports Migration from non-APPC app to APPC app via App View.
   
   Because of SQLite limitations (alter table does not allow DROP) we had to temporarily create a new table, migrate data without appcUpgraded there and then delete the original updates data, renaming the temporary table to update in the end.

**Database changed?**

   Yes (102 -> 103)

**Where should the reviewer start?**

- [ ] AppsFragment.java

**How should this be manually tested?**

  Test updates flows in Apps. 
  See if listAppcUpgrades is not called anymore from workmanager
  Check if migration via app view still works correctly.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-79](https://aptoide.atlassian.net/browse/APP-79)

**Code Review Checklist**
